lib/community/pages/create_post_page.dart

```diff
@@ -22,8 +22,16 @@ import 'package:thunder/utils/instance.dart';
 class CreatePostPage extends StatefulWidget {
   final int communityId;
   final FullCommunityView? communityInfo;
+  final void Function(DraftPost? draftPost)? updateDraft;
```

Comment:
For the naming of this, I think I prefer `onUpdateDraft` to match similar naming conventions for callback functions!

Reply:
Done.